packages/components/src/components/multi-select/multi-select/multi-select.tsx

```diff
@@ -207,83 +202,73 @@ export class MultiSelect {
-              </span>
-            )}
-          </label>
-          <div class={{ 'input-container': true, disabled: this.disabled }} ref={(el) => (this.inputContainer = el)}>
-            <input
-              role="combobox"
-              placeholder={getSelectedOptionsString(this.multiSelectOptions) || null}
-              autoComplete="off"
-              disabled={this.disabled}
-              required={this.required}
-              onInput={this.onInputChange}
-              onClick={this.onInputClick}
-              onKeyDown={this.onInputKeyDown}
-              ref={(el) => (this.inputElement = el)}
-              aria-invalid={this.state === 'error' ? 'true' : null}
-              {...getFilterInputAriaAttributes(
-                this.isOpen,
-                this.required,
-                labelId,
-                `${descriptionId} ${optionsSelectedId} ${messageId}`,
-                dropdownId
-              )}
-            />
-            {this.currentValue.length > 0 && (
-              <PrefixedTagNames.pButtonPure
-                class="icon reset-icon"
-                icon="close"
-                hideLabel="true"
-                theme={this.theme}
-                color={this.disabled ? 'state-disabled' : 'primary'}
-                onClick={this.onResetClick}
-                onKeyDown={(e) => e.key === 'Tab' && (this.isOpen = false)}
-                disabled={this.disabled}
-              >
-                Reset selection
-              </PrefixedTagNames.pButtonPure>
+      <div class="root">
+        <Label
+          host={this.host}
+          label={this.label}
+          description={this.description}
+          htmlFor={inputId}
+          isRequired={this.required}
+          isDisabled={this.disabled}
+        />
+        {/* in case, sr-only text is not placed here then the clear button is not able to focus the input for unknown reasons */}
+        {this.currentValue.length > 0 && (
+          <span id={optionsSelectedId} class="sr-only">
+            {getSelectedOptions(this.multiSelectOptions).length} options selected
+          </span>
+        )}
+        <div class={{ wrapper: true, disabled: this.disabled }} ref={(el) => (this.inputContainer = el)}>
+          <input
+            id={inputId}
+            role="combobox"
+            placeholder={getSelectedOptionsString(this.multiSelectOptions) || null}
+            autoComplete="off"
+            disabled={this.disabled}
+            required={this.required}
+            onInput={this.onInputChange}
+            onClick={this.onInputClick}
+            onKeyDown={this.onInputKeyDown}
+            ref={(el) => (this.inputElement = el)}
+            aria-invalid={this.state === 'error' ? 'true' : null}
+            {...getFilterInputAriaAttributes(
+              this.isOpen,
+              this.required,
+              htmlLabelId,
+              `${htmlDescriptionId} ${optionsSelectedId} ${htmlMessageId}`,
```

Comment:
Why doesn't the middle one have a `html` prefix? Or why do the other two have one?

Reply:
I've removed the html prefix.